scripts/src/main/resources/scripts/command/build

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
```

Comment:
as this line is already included in `functions` that is sourced one line after, why do we need to add it redundantly to all scripts sourcing `functions`?
Will not really hurt but seems pointless clutter to me.

Reply:
As already mentioned in my PR comment. I considered the alternative to only start tracing within _function_ -  but decided against it, since this makes reading the trace log out more difficult that it already is. Imho it should trace all executed lines - and not skip some important context switches just because some may consider the related code as pointless clutter.

Imagine a stacktrace, where you skip some function call - makes not much sense to me on TRACE level.

There could even be errors in the call of _functions_ itself. Or imagine the _functions_ file were within a new commandlet called - not sourced. This would stop tracing at all on return if the trace flags have not been activated in the original calling shell.

Sure - the PR is a poor mans tracing. But it is a quick win - even with the complete solution, which adds the trace on detection at each script begin.

Reply:
OK, I understood. I was not aware that this actually makes a difference in the resulting trace log.
So I just close this as resolved. Thanks for clarification.